packages/governance/test/unitTests/test-multiPluralityBallotCount.js

```diff
@@ -0,0 +1,291 @@
+
+  const positions = template.getDetails().positions;
+  t.deepEqual(positions.length, 5);
+
+  E(creatorFacet).submitVote(aliceSeat, [positions[0]], 2n);
+  E(creatorFacet).submitVote(bobSeat, [positions[2]], 15n);
+  E(creatorFacet).submitVote(tedSeat, [positions[1]], 27n);
+  E(creatorFacet).submitVote(carolSeat, [positions[3]], 17n);
+  await E(creatorFacet).submitVote(jackSeat, [positions[4]], 15n);
+  closeFacet.closeVoting();
+
+  const outcome = await E(publicFacet).getOutcome();
+  t.assert(outcome.length === 3);
+  t.deepEqual(outcome[0], BEEF);
+  t.deepEqual(outcome[1], CHICKEN);
+  t.assert(outcome[2].text === PORK.text || outcome[2].text === DUCK.text);
+
+  await eventLoopIteration();
+  t.like(storageRoot.getBody('mockChainStorageRoot'), {
+    outcome: 'win',
+    position: outcome,
+  });
+});
+
+test('single plurality spoiled', async t => {
+  const questionSpec = coerceQuestionSpec({
+    method: ChoiceMethod.PLURALITY,
+    issue: SIMPLE_ISSUE,
+    positions: [FISH, BEEF, PORK],
+    electionType: ElectionType.SURVEY,
+    maxChoices: 1,
+    maxWinners: 2,
+    closingRule: FAKE_CLOSING_RULE,
+    quorumRule: QuorumRule.MAJORITY,
+    tieOutcome: BEEF,
+  });
+
+  const { publisher } = makePublisherFromFakes();
+  const { publicFacet, creatorFacet } = makeMultiPluralityVoteCounter(
+    questionSpec,
+    0n,
+    FAKE_COUNTER_INSTANCE,
+    publisher,
+  );
+  const aliceTemplate = publicFacet.getQuestion();
+  const aliceSeat = makeHandle('Voter');
+
+  const alicePositions = aliceTemplate.getDetails().positions;
+  t.deepEqual(alicePositions.length, 3);
+  t.deepEqual(alicePositions[0], FISH);
+
+  await t.throwsAsync(
+    () => E(creatorFacet).submitVote(aliceSeat, [harden({ text: 'no' })]),
+    {
+      message: `The specified choice is not a legal position: {"text":"no"}.`,
+    },
+  );
+});
+
+test('single plurality revote', async t => {
+  const questionSpec = coerceQuestionSpec({
+    method: ChoiceMethod.PLURALITY,
+    issue: SIMPLE_ISSUE,
+    positions: [FISH, BEEF, PORK],
+    electionType: ElectionType.SURVEY,
+    maxChoices: 1,
+    maxWinners: 2,
+    closingRule: FAKE_CLOSING_RULE,
+    quorumRule: QuorumRule.NO_QUORUM,
+    tieOutcome: BEEF,
+  });
+
+  const { publisher, storageRoot } = makePublisherFromFakes();
+  const { publicFacet, creatorFacet, closeFacet } =
+    makeMultiPluralityVoteCounter(
+      questionSpec,
+      0n,
+      FAKE_COUNTER_INSTANCE,
+      publisher,
+    );
+  const template = publicFacet.getQuestion();
+  const aliceSeat = makeHandle('Voter');
+  const bobSeat = makeHandle('Voter');
+
+  const positions = template.getDetails().positions;
+
+  E(creatorFacet).submitVote(aliceSeat, [positions[0]], 23n);
+  E(creatorFacet).submitVote(bobSeat, [positions[1]], 15n);
+  await E(creatorFacet).submitVote(bobSeat, [positions[2]], 47n);
+  closeFacet.closeVoting();
+
+  const outcome = await E(publicFacet).getOutcome();
+  t.deepEqual(outcome, [PORK, FISH]);
```

Comment:
With maxWinners at 2, this test won't show that Bob's first vote was ignored.  Please change maxWinners to 3, so that we can tell that winningPositions only had two entries.